crates/next-dev/benches/util/mod.rs

```diff
@@ -206,6 +207,10 @@ impl<'a, 'b, A: AsyncExecutor> AsyncBencherExtension for AsyncBencher<'a, 'b, A,
             config.as_deref(),
             None | Some("") | Some("no") | Some("false")
         );
+        let log_progress = !matches!(
+            std::env::var("TURBOPACK_BENCH_PROGRESS").ok().as_deref(),
+            None | Some("") | Some("no") | Some("false")
```

Comment:
Move this logic into an `env_flag("TURBOPACK_BENCH_PROGRESS") -> bool` function.